### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by knae11** [woowacourse-teams/2021-gpu-is-mine#355] [General]
# 기능

- 회원가입
- 로그인
- gpu 서버 관리
- job 관리
- 이메일 알림

# Asserts
- zip

<hr /><em>This discussion was created from the release <a href='https://github.com/woowacourse-teams/2021-gpu-is-mine/releases/tag/'>v1.0.0</a>.</em>

**Comment by sjpark-dev** [comment]
비밀번호 암호화
로그 표시 및 시각화

**Reply by knae11** [reply]
상세 기능도 넣어주는게 좋을까? 이부분이 고민되뮤ㅠㅠ 처음에 상세 내용까지 넣을까하다가 큰 기능위주로 넣긴 했는데 음 어느정도 넣는게 좋을지 모르겠엏ㅎㅎㅎ

**Reply by bigsaigon333** [reply]
1.0.0 이고, 처음 남기는거라서 최대한 자세하게 남겨놓으면 좋지 않을까요?

**Comment by bigsaigon333** [comment]
1. zip 파일은 굳이 안해도 되지 않을까요..? 그냥 github repo tag로 관리되어도 괜찮지 않을까...

2. 기술스택을 FE, BE, Infra 뭐 이런식으로 나누는건 어떨까요? FE 기술스택은 추가로 기재하겠습니다
